presto-hive/src/test/java/com/facebook/presto/hive/TestHivePushdownFilterQueries.java

```diff
@@ -626,6 +626,12 @@ public void testStructs()
         assertQueryUsingH2Cte("SELECT info.orderkey FROM lineitem_ex WHERE info IS NULL", rewriter);
         assertQueryUsingH2Cte("SELECT info.orderkey FROM lineitem_ex WHERE info IS NOT NULL", rewriter);
 
+        // verify that only the required sub-fields are read when the struct is not projected and there is a top level not null on it.
```

Comment:
- this comment is misleading as these tests doesn't verify that `only the required sub-fields are read`; these tests only ensure that the queries succeeds and produce correct results
- let's remove this comment and move the queries as follows:
  - #1 and #2 into `// filter-only struct` section
  - #3 next to `info [NOT] IS NULL` queries on L627-628
  - #4 next to `orderkey < 1000` on L625
- SQL keywords need to be capitalized: `is not null` -> `IS NOT NULL`